### PR TITLE
Code cleanup

### LINKED ISSUE #189: Refactor Casbin Sequelize connection
### Task
The current setup requires the `dbUrl` to be sent which forces `logging: true`. We want to instead send username, password, and db, as part of a connection object that can set `logging: false`.

### Acceptance Criteria
- [x] Casbin connection uses object instead of connection string

### Notes
Related to this issue that I opened. https://github.com/node-casbin/sequelize-adapter/issues/25#issuecomment-639550266 _note they are not fixing it_

### LINKED ISSUE #195: Update terraform
### User Story
As a developer
I'd like to have updated terraform
So that I can be confident that any future deployments will match the current config

### Acceptance Criteria
- [x] The ASG sets EC2 instance size to `t2.micro`
- [x] The RDS size is `t3.small`
- [ ] DB URL is included in the task definition group as an environment variable
- [x] JWT key is a variable in TF

### LINKED ISSUE #198: npm test doesn't remove all test writes
**Describe the bug**
When running `npm test` there are values written to the database that is not removed upon test completion. Over time this clutters up a database with junk values. In particular, it looks like `Contacts` and `Entities` is effected, however, `EntityContacts` is not.

**To Reproduce**
1. run `npm test`
2. check the database for added values

**Expected behavior**
No values added from the test are present in the database after test conclusion.

### LINKED ISSUE #199: testing on a blank db causes failures
**Describe the bug**
When running `npm test` against a fresh db with no data there are invalid test failures.

**To Reproduce**
1. create a fresh db by running `npm run db-create`
2. run `npm test`

**Expected behavior**
A clear database will pass all tests the same as a populated database will

### LINKED ISSUE #200: Project website
### User Story
As a potential user or contributor of bmore responsive
I'd like to have a project website
So that I can clearly understand how to use and contribute to the project

### Acceptance Criteria
- [x] mkdocs add to the project
- [ ] github page setup to use mkdocs

### LINKED ISSUE #201: Refactor Casbin db connection to remove SQL from logs
### Task
Currently, the Casbin db connection defaults to a robust logging. We do not need this level of detail in the logs for production. 

### Acceptance Criteria
- [x] Casbin db connection implements options

### PR DESCRIPTION
**Pull request by revjtanton**
# Description

Just some cleanup and misc. code work before starting on real tickets.
resolves #189 
resolves #195 
resolves #198 
resolves #199 
resolves #200 
resolves #201 

## Todos

- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by revjtanton** [comment]
@all-contributors add @mpiroc for infra

File: .travis.yml [hunk 1]
```diff
@@ -5,13 +5,14 @@ install:
 - npm install
 - npm install -g codecov
 - npm install -g nyc
+- mkdir site
```
Comment (c-w-allen) [inline .travis.yml:8]: I personally like abstracting the CICD file to just run make commands. That way you are more adjusting only the stages themselves in this file. Just something to think about later I suppose.

File: src/index.js [hunk 2]
```diff
@@ -1,90 +1,97 @@
...
+const app = express()
 const swaggerOptions = {
 	customCss: '.swagger-ui .topbar { display: none }'
-};
+}
 const apiLimiter = rateLimit({
 	windowMs: 15 * 60 * 1000, // 15 minutes
-	max: 100
-});
+	max: 1000
```
Comment (c-w-allen) [inline src/index.js:23]: What was the motivation for increasing this?

File: src/routes/csv.js [hunk 3]
```diff
@@ -1,37 +1,34 @@
...
+      response.setMessage("Model type is invalid")
     }
+  } catch (e) {
+    console.error(e)
+    code = 500
+  }
 
-    return utils.response(res, code, message);
-});
+  return res.status(response.getCode()).send(response.getMessage())
```
Comment (c-w-allen) [inline src/routes/csv.js:31]: I see what you're doing here, but I can't help but question if there's a simpler way to utilize the `res` variable

File: src/routes/user-role.js [hunk 4]
```diff
@@ -24,75 +23,79 @@ router.get('/', async (req, res) => {
...
+			
+			const e = await utils.loadCasbin()
+			const p = [role, path, method]
+			removed = await e.removePolicy(...p)
 
-		if (removed) {
-			code = 200;
-			message = 'policy deleted';
+			if (removed) {
+
```
Comment (c-w-allen) [inline src/routes/user-role.js:85]: extra space

File: src/routes/user.js [hunk 5]
```diff
@@ -1,262 +1,254 @@
...
-const router = new Router();
-const max = (process.env.NODE_ENV !== 'production') ? 50000 : 5;
+import { Router } from 'express'
+import rateLimit from 'express-rate-limit'
+import validator from 'validator'
+import utils from '../utils'
+import email from '../email'
+
+const router = new Router()
+const max = (process.env.NODE_ENV !== 'production') ? 50000 : 5
```
Comment (c-w-allen) [inline src/routes/user.js:8]: nit: can we give `max` a more specific name?

File: .travis.yml (hunk 1 above)
Comment (revjtanton) [reply .travis.yml:8]: For sure. This was a test of mixing the mkdocs with `npm run`, put that in as a placeholder. Forgot to remove it.

File: src/index.js (hunk 2 above)
Comment (revjtanton) [reply src/index.js:23]: Also a test with the mkdocs. With it at only 100 and serving mkdocs it hit a limit with just a few page loads.

File: src/routes/csv.js (hunk 3 above)
Comment (revjtanton) [reply src/routes/csv.js:31]: This is setting it up to be swapped to how we'll handle Lambda later. I'm angling towards a unified response that can support both a Express/Docker setup and a Lambda setup.

File: src/routes/user.js (hunk 5 above)
Comment (revjtanton) [reply src/routes/user.js:8]: I changed how it's used, so `max` is the best name here.

**Pull request merged**